core/src/main/java/tech/tablesaw/joining/DataFrameJoiner.java

```diff
@@ -434,42 +487,62 @@ public Table rightOuter(boolean allowDuplicateColumnNames, Table... tables) {
      * @param table2   The table to join with
      * @param col2Name The column to join on. If col2Name refers to a double column, the join is performed after
      *                 rounding to integers.
+     * @return The resulting table
      */
     public Table rightOuter(Table table2, String col2Name) {
-        return rightOuter(table2, col2Name, false);
+        return rightOuter(table2, false, col2Name);
     }
 
     /**
-     * Joins the joiner to the table2, using the given column for the second table and returns the resulting table
+     * Joins the joiner to the table2, using the given columns for the second table and returns the resulting table
+     *
+     * @param table2    The table to join with
+     * @param col2Names The columns to join on. If a name refers to a double column, the join is performed after
+     *                  rounding to integers.
+     * @return The resulting table
+     */
+    public Table rightOuter(Table table2, String[] col2Names) {
+        return rightOuter(table2, false, col2Names);
+    }
+
+    /**
+     * Joins the joiner to the table2, using the given columns for the second table and returns the resulting table
      *
      * @param table2   The table to join with
-     * @param col2Name The column to join on. If col2Name refers to a double column, the join is performed after
-     *                 rounding to integers.
      * @param allowDuplicateColumnNames if {@code false} the join will fail if any columns other than the join column have the same name
      *                                  if {@code true} the join will succeed and duplicate columns are renamed
+     * @param col2Names The columns to join on. If a name refers to a double column, the join is performed after
+     *                 rounding to integers.
+     * @return The resulting table
      */
 
-    public Table rightOuter(Table table2, String col2Name, boolean allowDuplicateColumnNames) {
-        Table leftOuter = table2.join(col2Name).leftOuter(table, column.name(), allowDuplicateColumnNames);
+    public Table rightOuter(Table table2, boolean allowDuplicateColumnNames, String... col2Names) {
+        Table leftOuter = table2.join(col2Names).leftOuter(table, allowDuplicateColumnNames, columnNames);
 
         // reverse the columns
         Table result = Table.create(leftOuter.name());
+        // loop on table that was originally first (left) and add the left-joined matching columns by name
         for (String name : table.columnNames()) {
-            result.addColumns(leftOuter.column(name));
+            try {
+                result.addColumns(leftOuter.column(name));
+            } catch (IllegalStateException e) {
+                System.out.println("NOTE: DataFrameJoiner.rightOuter(): skipping left table's column,'"
```

Comment:
We should either throw an exception here or do logging with slf4j rather than printing to std out

Reply:
I don't yet see any use of slf4j in this project.  Would this be the first use?  If so, is log4j preferred over logback?  I could introduce either one but I've gotten the impression that logback is gaining over log4j in popularity.